src/game/ingame_menu.c

```diff
@@ -1369,6 +1367,7 @@ void print_peach_letter_message(void) {
     void **dialogTable;
     gInGameLanguage = eu_get_language();
     switch (gInGameLanguage) {
+        default:
```

Comment:
This is valid, but bad practice to put `default` somewhere other than the end of a switch case.